msteams-platform/toolkit/provision.md

```diff
@@ -222,6 +222,9 @@ You can add following configuration snippet to `.fx\configs\config.{env}.json` f
 
 After adding the snippet, add your client secret to related environment variable for Teams Toolkit to resolve the actual client secret during provision.
 
+> [!NOTE]
+> A Teams Toolkit generated project has a pre-defined set of Visual Studio Code debugging tasks in the `.vscode/tasks.json` file. These pre-defined tasks are generated by Teams Toolkit version 4.1.0 or later. Add the configuration snippet in the `.vscode/tasks.json` file.
+
```

Comment:
I think the key point here is to mention use `.vscode/tasks,json` for customizing debugging tasks (such as use existing bot) and use `.fx\configs\config.{env}.json` is for cloud environments.